docs/rust.md

```diff
@@ -1,3 +1,33 @@
 # Awesome Open Source - Rust
 
-Coming Soon!
+- [actix-web](https://github.com/actix/actix-web) - Actix Web is a powerful, pragmatic, and extremely fast web framework for Rust.
+- [alacritty](https://github.com/alacritty/alacritty) — A cross-platform, GPU enhanced terminal emulator
+- [Asymmetrik/ngx-deploy-azure](https://github.com/Asymmetrik/ngx-deploy-azure) - Deploy Angular apps to Azure.
+- [bat](https://github.com/sharkdp/bat) - A cat(1) clone with syntax highlighting and Git integration.
+- [chrono](https://github.com/chronotope/chrono) - Date and time library for Rust.
+- [clap](https://github.com/clap-rs/clap) - A simple to use, full-featured Command Line Argument Parser for Rust.
+- [diesel](https://github.com/diesel-rs/diesel) - A safe, extensible ORM and Query Builder for Rust.
+- [dioxus](https://github.com/DioxusLabs/dioxus) - Elegant React-like library for building user interfaces for desktop, web, mobile, and more.
+- [fake-rs](https://github.com/cksac/fake-rs) - A library for generating fake data.
+- [Fuchsia OS](https://fuchsia.dev/) - A new open source operating system created at Google.
+- [glium](https://github.com/glium/glium) - Safe OpenGL wrapper for the Rust language.
+- [google/benchmark](https://github.com/google/benchmark) - A microbenchmark support library.
```

Comment:
this is a c++ package

Reply:
Removed! Read about it in a article measuring rust speeds so assumed it was in rust. My bad.

Reply:
no worries, thanks sir!